create_new_plugin/create_new_plugin.md

```diff
@@ -1,5 +1,3 @@
-# Create your own plugin
-
```

Comment:
When it comes to single chapter tutorials, don't they need the `# <Title` on top of the chapter anymore? If so, I will need to change the tutorials contribution guide :D

Reply:
We change that because now we are getting chapter titles from index file, and a h1 is created from it That's why we're skipping titles everywhere and starting from h2 subtitles